tests/testthat/test_distance_function.R

```diff
@@ -1,9 +1,9 @@
 context("Testing computation of distance")
-
+source("../permfun.R")
```

Comment:
I think this line would be unnecessary if `permutations()` is transferred to R/misc.R.

Reply:
Good point!